deployment/src/main/java/io/quarkiverse/asyncapi/generator/QuteTemplateHelper.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkiverse.asyncapi.generator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.eclipse.microprofile.config.Config;
+
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.Template;
+
+public class QuteTemplateHelper {
+
+    private static Engine engine = Engine.builder().addDefaults().build();
```

Comment:
Not sure this is correct. You can `@Inject Engine` or lookup from CDI

Reply:
This, like its open api generator counterpart, it is used in CodeGen step, which does not support CDI. Thats why the engine is created programatically rather than injecting it.